packages/uicore/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -40,6 +40,7 @@ export interface MultiSelectProps
     | 'onActiveItemChange'
   > {
   itemRender?: Props['itemRenderer']
+  avoidResetOnSelect?: boolean // This will prevent scroll reset to top
```

Comment:
Can we add proper JS-Doc style syntax, as that would be suggested when consuming the prop. This way it won't be picked by intellisense.

```
 /** avoid resetting the scroll to the top upon selection. */
  avoidResetOnSelect?: boolean
  ```
  
![image](https://github.com/harness/uicore/assets/100121280/608aaad6-b71e-45c0-8b95-f30c763be9c7)

Reply:
will change comment format while making this as default.